docs/src/"troubleshooting.md"

```diff
@@ -0,0 +1,24 @@
+# Troubleshooting
```

Comment:
Let's put this into the howto.md rather than troubleshooting.md, under a subsection

```
## Troubleshooting connection issues 
```